configure

```diff
@@ -648,15 +648,15 @@ fi
 #
 if [ -n "$BUILD_API_DOCS" ]; then
 	echo "Building API documentation via doxygen and graphviz."
-	printf "Checking for doxygen ..."
+	printf "Checking for doxygen... "
```

Comment:
I took the liberty of adding what seemed to be a missing space here, hope you all don't mind.